### DISCUSSION TITLE
Routing verwirrt

### DISCUSSION THREAD

**Discussion by EisenbahnfotografWesterwamd** [Traewelling/traewelling#2811] [Questions / Support] [answered]
### Describe the bug

Wenn man in manche züge eincheckt und das meist auf weitere Strecken wird immer mehr komische Abzweigungen angezeigt... Um es besser zu verstehen ist ein Bild dabei gefügt und dazu wenn ich selbst ein check in mache und ein Routing gefunden wird ist das Routing perfekt wirklich sehr sehr perfekt keine Abzweigungen oder sonst was...
![Screenshot_20240715_111134_Chrome](https://github.com/user-attachments/assets/6eb5d8bf-6289-4a03-9746-34f7f2e3f6bb)

### Steps to reproduce

1.station auswählen. 2. Irgendein Zug auswählen. 3. Auf der Karte gucken und da erkennt man es dann

**Comment by MrKrisKrisu** [comment, marked as answer]
Wie diese Kartendarstellung zustande kommt, haben wir hier erklärt.
->https://help.traewelling.de/features/map/

Wenn du noch Fragen hast, melde dich einfach wieder.
